homework_4/css/style.css

```diff
@@ -0,0 +1,179 @@
+body {
+	background-image: url(../img/background.jpg);
+}
+
+.box {
+	width: 1000px;
+	height: 998px;
+	margin: 38px 129px 30px 151px;
+	background: white;
+	font-family: Tahoma , Arial , sans-serif;
+}
+
+/*HEADER*/
+
+.header__logo {
+	margin-left: 28px;
+	margin-top: 11px;
+	float: left;
+}
+
+.header__contact {
+	float: right;
+	margin-top: 11px;
+	margin-right: 34px;
+	color: #5fa5bb;
+	line-height: 18px;
+}
+
+.header__contact__item:first-child {
+	font-weight: bold;
+	line-height: 21px;
+}
+
+.header__countries {
+	height: 50px;
+	width: 350px;
+	font-size: 10px;
+	padding-top: 111px;
+	padding-left: 630px;
```

Comment:
use float right instead of calculating paddings, you`ll have a problem with placing this el, you can make a container for it,  which will devide width of header with logo, and locate this block in it